source/MaterialXRenderGlsl/GlslRenderer.cpp

```diff
@@ -27,19 +27,20 @@ const float FAR_PLANE_PERSP = 100.0f;
 // GlslRenderer methods
 //
 
-GlslRendererPtr GlslRenderer::create(unsigned int width, unsigned int height, Image::BaseType baseType)
+GlslRendererPtr GlslRenderer::create(unsigned int width, unsigned int height, Image::BaseType baseType, const std::string &extension)
```

Comment:
Instead of modifying the low-level constructor for GlslRenderer, I would recommend using the existing method TextureBaker::setExtension, which can be found here:

https://github.com/materialx/MaterialX/blob/main/source/MaterialXRenderGlsl/TextureBaker.h#L44